downstream/modules/platform/proc-controller-image-pull-policy.adoc

```diff
@@ -0,0 +1,26 @@
+[id="proc_controller-image-pull-policy_{context}"]
+
+= controller-image-pull-policy
+
+
+. Under *Image Pull Policy*, click on the radio button to select *Always*, *Never*, or *IfNotPresent*.
+. Define the *Image Pull Secrets*
+.. Click on the angle bracket to show the *Add Image Pull Secret*.
+.. Click on the *Add Image Pull Secret* and enter the *Value* in the field.
+. Define the *Web container resource requirements* by clicking on the angle brackets to display the *Web container resource requirements* fields.
+.. Enter the necessary values for *Limits*, and *Requests*.
```

Comment:
```suggestion
.. Under *Limits*, and *Requests*, enter values for *CPU cores*, *Memory*, and *Storage*.
```
Just a suggestion to be specific and exact about the fields that users are required to enter.

Reply:
Users are not _required_ to enter limits or requests here.  There are default values which will work in most cases. 

I am not sure if there is a way that we denote if a field is optional or not, but the only "required" field for a simple AutomationController deployment is the "name" field.